### PR TITLE
feat(tabs): add express theme & additional variations

### PR DESCRIPTION
**Pull request by pfulton**
## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

### CODE REVIEW

**Comment by Westbrook** [comment]
@pfulton can we get https://github.com/adobe/spectrum-css/pull/1176/files#diff-4fb35d35e5e57cd80a2251751d55932eb9181f17c857e414d27c72c39f8bb165 added to this PR?

**Comment by Westbrook** [comment]
Oh look it's already here: https://github.com/adobe/spectrum-css/pull/1323 😅 Would love to see these land together!

**Comment by pfulton** [comment]
@Westbrook yeah, I think we can [make that happen](https://github.com/adobe/spectrum-css/pull/1355/commits/da81a6e844a0348c30ee4408ca297d48e174953d) 😀

File: components/tabs/skin.css [hunk 1]
```diff
@@ -11,76 +11,198 @@ governing permissions and limitations under the License.
...
+    --spectrum-tabs-m-textonly-divider-background-color: transparent;
+    --spectrum-tabs-m-textonly-tabitem-selection-indicator-background-color: transparent;
+    --spectrum-tabs-m-textonly-tabitem-text-color-disabled: GrayText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-hover: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-key-focus: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color-selected: ButtonText;
+    --spectrum-tabs-m-textonly-tabitem-text-color: ButtonText;
+    --spectrum-tabs-m-vertical-rule-color: transparent;
+    --spectrum-tabs-m-quiet-textonly-divider-background-color: transparent;
+    --spectrum-tabs-m-quiet-textonly-tabitem-selection-indicator-background-color: transparent;
```
Comment (jnurthen) [inline components/tabs/skin.css:137]: The variables don't exist so WHCM is not working correctly. Suggest the following:
```
    --spectrum-tabs-emphasized-texticon-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-emphasized-texticon-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-emphasized-texticon-tabitem-text-color-selected: HighlightText;
    --spectrum-tabs-emphasized-textonly-divider-background-color: transparent;
    --spectrum-tabs-emphasized-textonly-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-quiet-textonly-divider-background-color: transparent;
    --spectrum-tabs-quiet-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-quiet-textonly-tabitem-selection-indicator-background-color: transparent;
    --spectrum-tabs-textonly-divider-background-color: transparent;
    --spectrum-tabs-textonly-tabitem-focus-ring-border-color-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-disabled: GrayText;
    --spectrum-tabs-textonly-tabitem-icon-color-hover: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-selected-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-icon-color-selected: HighlightText;
    --spectrum-tabs-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
    --spectrum-tabs-textonly-tabitem-text-color-disabled: GrayText;
    --spectrum-tabs-textonly-tabitem-text-color-hover: ButtonText;
    --spectrum-tabs-textonly-tabitem-text-color-selected-key-focus: ButtonText;
    --spectrum-tabs-textonly-tabitem-text-color-selected: HighlightText;
    --spectrum-tabs-textonly-tabitem-text-color: ButtonText;
    --spectrum-tabs-vertical-quiet-textonly-divider-background-color: transparent;
    --spectrum-tabs-vertical-rule-color: transparent;
    --spectrum-tabs-vertical-textonly-divider-background-color: transparent;
    --spectrum-tabs-vertical-textonly-tabitem-selection-indicator-background-color-selected: Highlight;
  ```

Comment (pfulton) [reply components/tabs/skin.css]: @jnurthen thanks so much for those suggestions. I just pushed a commit with those vars and took a quick look in AssistivLabs, and things look ok to me. Do you mind taking another look and letting me know if we're good to go?

Comment (jnurthen) [reply components/tabs/skin.css]: LGTM

**Review by jnurthen — APPROVED** [review]
WHCM LGTM

File: components/tabs/skin.css (hunk 1 above)
Comment (pfulton) [reply components/tabs/skin.css]: Thanks!

**Pull request merged**